### PR TITLE
add keypoints to 05

### PR DESCRIPTION
**Pull request by wrightaprilm**
adding keypoints to 05 for lesson release

### CODE REVIEW

**Comment by wrightaprilm** [comment]
We good on this, @maxim-belkin?

File: _episodes/05-merging-data.md [hunk 1]
```diff
@@ -11,7 +11,10 @@ objectives:
     - "Employ `to_csv` to export a DataFrame in CSV format."
     - "Join DataFrames using common fields (join keys)."
 keypoints:
-    - "FIXME"
+    - "`Merge` and `concat` can be used to recombine subsets from a DataFrame, or even data from different files."
```
Comment (maxim-belkin) [inline _episodes/05-merging-data.md]: Python is case-sensitive, so... maybe update to:
> "Panda's `.merge` and `.concat` methods are used to (re-)combine subsets of a DataFrame or even data from different files."

File: _episodes/05-merging-data.md [hunk 2]
```diff
@@ -11,7 +11,10 @@ objectives:
...
+    - "DataFrames can be combined using join keys with the `join` function."
+    - "Joining two DataFrames can be done in multiple ways (left, right, and inner) according to what data must be in the final DataFrame."
+    - "`to_csv` can be used to write out DataFrames in CSV format."
```
Comment (maxim-belkin) [inline _episodes/05-merging-data.md]: I suggest spelling out CSV: Comma-Separated Values format

File: _episodes/05-merging-data.md [hunk 3]
```diff
@@ -11,7 +11,10 @@ objectives:
     - "Employ `to_csv` to export a DataFrame in CSV format."
     - "Join DataFrames using common fields (join keys)."
 keypoints:
-    - "FIXME"
+    - "`Merge` and `concat` can be used to recombine subsets from a DataFrame, or even data from different files."
+    - "DataFrames can be combined using join keys with the `join` function."
+    - "Joining two DataFrames can be done in multiple ways (left, right, and inner) according to what data must be in the final DataFrame."
```
Comment (maxim-belkin) [inline _episodes/05-merging-data.md]: according to -> depending on

**Comment by maxim-belkin** [comment]
I consider only the first comment (about case-sensitivity) to be important. The other two are minor

File: _episodes/05-merging-data.md (hunk 1 above)
Comment (wrightaprilm) [reply _episodes/05-merging-data.md]:
```suggestion
    - "`merge` and `concat` can be used to (re)-combine subsets from a DataFrame, or even data from different files."
```

I left them in without the dot notation, but lowercase. There are other points where we refer to methods without dot notation.

File: _episodes/05-merging-data.md (hunk 3 above)
Comment (wrightaprilm) [reply _episodes/05-merging-data.md]:
```suggestion
    - "Joining two DataFrames can be done in multiple ways (left, right, and inner) depending on what data must be in the final DataFrame."
```

File: _episodes/05-merging-data.md (hunk 2 above)
Comment (wrightaprilm) [reply _episodes/05-merging-data.md]: We define CSV in 02, so I think it's OK to leave.

File: _episodes/05-merging-data.md (hunk 1 above)
Comment (maxim-belkin) [reply _episodes/05-merging-data.md]: Yeah, I'm fine with that. I think it _might_ be useful to use the same approach across multiple lessons but this is not important at this point.

Comment (maxim-belkin) [reply _episodes/05-merging-data.md]: shall it, perhaps, be `combine subsets of a DataFrame` ?

Comment (wrightaprilm) [reply _episodes/05-merging-data.md]:
```suggestion
    - "`Merge` and `concat` can be used to combine subsets of a DataFrame, or even data from different files."
```

File: _episodes/05-merging-data.md [hunk 4]
```diff
@@ -11,7 +11,10 @@ objectives:
     - "Employ `to_csv` to export a DataFrame in CSV format."
     - "Join DataFrames using common fields (join keys)."
 keypoints:
-    - "FIXME"
+    - "Pandas' `merge` and `concat` can be used to (re)-combine subsets from a DataFrame, or even data from different files."
```
Comment (wrightaprilm) [inline _episodes/05-merging-data.md]:
```suggestion
    - "Pandas' `merge` and `concat` can be used to combine subsets of a DataFrame, or even data from different files."
```

**Comment by wrightaprilm** [comment]
OK, made that last change.

I don't think it would be bad to revisit how we refer to methods, and if we're doing it consistently in the future. Would be a good instructor training checkout exercise.

File: _episodes/05-merging-data.md [hunk 5]
```diff
@@ -11,7 +11,10 @@ objectives:
     - "Employ `to_csv` to export a DataFrame in CSV format."
     - "Join DataFrames using common fields (join keys)."
 keypoints:
-    - "FIXME"
+    - "Pandas' `merge` and `concat` can be used to combine subsets of a DataFrame, or even data from different files."
+    - "DataFrames can be combined using join keys with the `join` function."
```
Comment (maxim-belkin) [inline _episodes/05-merging-data.md]: "join keys" seems unclear / jargonny

File: _episodes/05-merging-data.md (hunk 3 above)
Comment (maxim-belkin) [reply _episodes/05-merging-data.md]: Did you decide to not include this one?

File: _episodes/05-merging-data.md (hunk 5 above)
Comment (wrightaprilm) [reply _episodes/05-merging-data.md]:
```suggestion
    - "DataFrames can be combined using based on column names with the `join` function."
```

Comment (wrightaprilm) [reply _episodes/05-merging-data.md]: like so?

File: _episodes/05-merging-data.md (hunk 2 above)
Comment (wrightaprilm) [reply _episodes/05-merging-data.md]: Forgot to click 'commit'

File: _episodes/05-merging-data.md (hunk 5 above)
Comment (maxim-belkin) [reply _episodes/05-merging-data.md]: Maybe... alternative:
> `join` function combines DataFrames based on index or a column.

Comment (wrightaprilm) [reply _episodes/05-merging-data.md]: I like that!

**Pull request merged**